build-tools/packages/build-tools/src/common/fluidRepo.ts

```diff
@@ -33,6 +36,9 @@ export interface PolicyConfig {
 export interface IFluidRepoPackage {
     directory: string;
     ignoredDirs?: string[];
+    branchReleaseTypes?: {
```

Comment:
Docs :) What does it mean for this to be undefined? Just that nothing will be generated in the package.json file?